types/zxcvbn/index.d.ts

```diff
@@ -1,211 +1,224 @@
 // Type definitions for zxcvbn 4.4
 // Project: https://github.com/dropbox/zxcvbn#readme
 // Definitions by: Matt Traynham <https://github.com/mtraynham>
+//                 Muhun Kim <https://github.com/x86chi>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-export as namespace zxcvbn;
-
-export = zxcvbn;
```

Comment:
This needs `export =` instead of `export default`. Refer to https://github.com/DefinitelyTyped/DefinitelyTyped#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default